ttnn/cpp/ttnn/operations/sliding_window/sliding_window.hpp

```diff
@@ -43,19 +43,21 @@ struct SlidingWindowConfig {
     uint32_pair_t pad_hw_;
     uint32_pair_t dilation_hw_;
 
+    uint32_t groups;
```

Comment:
Why do we need `groups` here? Isn't it conv specific and not sliding window?

Reply:
Groups was an argument that was passed via conv_params. As sliding_window replaced conv_params, I thought the easiest approach would be to also add groups to sliding_window.

Reply:
But is "easiest" - the answer? Isn't it confusing from user standpoint?

Reply:
Removed `groups` from `sliding_window_config`